Gir.toml

```diff
@@ -11,35 +11,68 @@ single_version_file = true
 
 generate = [
     "Atk.Action",
-    # "Atk.Attribute",
-    # "Atk.AttributeSet",
     "Atk.Component",
+    "Atk.CoordType",
     "Atk.Document",
+    "Atk.EventListener",
+    "Atk.EventListenerInit",
+    "Atk.FocusHandler",
+    "Atk.GObjectAccessible",
+    "Atk.Hyperlink",
+    "Atk.HyperlinkImpl",
+    "Atk.HyperlinkStateFlags",
     "Atk.Hypertext",
     "Atk.Image",
+    # "Atk.Implementor",
+    # "Atk.KeyEventStruct",
     "Atk.Layer",
     "Atk.Misc",
+    # "Atk.NoOpObject",
+    "Atk.NoOpObjectFactory",
+    "Atk.ObjectFactory",
+    "Atk.Plug",
+    "Atk.PropertyChangeHandler",
+    # "Atk.PropertyValues",
     "Atk.Range",
     "Atk.Rectangle",
     "Atk.Registry",
+    "Atk.Relation",
+    "Atk.RelationType",
+    "Atk.RelationSet",
     "Atk.Role",
+    "Atk.Selection",
+    "Atk.Socket",
     "Atk.State",
-    "Atk.Table",
+    "Atk.StateType",
+    "Atk.StateSet",
+    "Atk.StreamableContent",
     "Atk.TableCell",
     "Atk.Text",
+    "Atk.TextAttribute",
+    "Atk.TextBoundary",
+    "Atk.TextClipType",
+    "Atk.TextGranularity",
+    "Atk.TextRange",
+    "Atk.Util",
+    "Atk.Value",
+    "Atk.ValueType",
+    "Atk.Window",
 ]
 
 ignore = [
-    "Atk.Relation",
-    "Atk.Plug", # conflicting implementations of trait `glib::IsA<glib::Object>` for type `auto::plug::Plug`
-    "Atk.Window", # conflicting implementations of trait `glib::translate::ToGlibPtr<'_, *mut gobject_ffi::GObject>` for type `auto::window::Window`
```

Comment:
IMHO "Atk.Plug" and "Atk.Window"'s traits need renaming to remove mixing with gdk's and gtk's

Reply:
Hm, there too many trait duplicates with gtk, now I don't sure that it need renaming.